### DISCUSSION TITLE
法学想run🇯🇵，很难。。。

### DISCUSSION THREAD

**Discussion by 120662366** [The-Run-Philosophy-Organization/run#290] [General]
目前大三下学期，很有润的打算，但根本无从落实，只有N2证书。有无润学先辈能指条明路？

**Comment by ghost** [comment]
转 IT

**Reply by ghost** [reply]
日文 N2 没有语言压力，你自己去看求人上 IT 什么要求，国内学了然后去了日本拿学签做相关打工インターン什么的，然后嘛，反正日本 IT 行业属于正常就労ビザ的行业，找到公司签了就win了。

或者你拿托福托业去念大学院。我估摸着要转专业才好做，这个就看你自己了。

**Comment by Alex-Jee** [comment]
> 目前大三下学期，很有润的打算，但根本无从落实，只有N2证书。有无润学先辈能指条明路？

文科类都很难吧，特别是法学这种对语言要求极高的，人家为啥不用自己本国学生呢？

**Reply by qibar** [reply]
财经类的非应届生，英语日语都行，家里中产，能否提供点思路？

**Reply by Alex-Jee** [reply]
不知道文商科要咋润，除了像nika那样去读个澳洲护理

**Reply by qibar** [reply]
谢谢，希望有其他看到这个问题的人回复我

**Comment by mitchellcuevas** [comment]
转码吧，学个java。

**Comment by Aranjuezzz** [comment]
新加坡或者澳洲jd，考bar后，在新加坡和澳洲执业，主要面向中国客户的ODI业务，实在不行做移民咨询也可以。
